### ISSUE TITLE
Unable to sell dogefather on pancakes finance V1 and V2

### ISSUE THREAD

**Issue by inshalayaz** [dogefatherBSC/dogefatherBSC#1]
![Screenshot_2021-05-08-07-41-37-71_08800057d42e3b72e31ae4fd4c41ba57](https://user-images.githubusercontent.com/52293580/117523262-58eec200-afd1-11eb-9103-03ef56591b65.jpg)
I'm getting this error when selling dogefather need help

**Comment by dogefatherBSC** [comment]
Have you try to remove decimal and adjust the slippage?
For quicker response, join our community on t.me/dogefatherbsc

**Comment by inshalayaz** [comment]
Yes I tried the slippage but not the decimal I will try that and come back

**Comment by inshalayaz** [comment]
![Screenshot_2021-05-08-13-23-43-84](https://user-images.githubusercontent.com/52293580/117532384-b9e1be80-b000-11eb-9994-19b2e0799eb5.jpg)

![Screenshot_2021-05-08-13-23-50-44_08800057d42e3b72e31ae4fd4c41ba57](https://user-images.githubusercontent.com/52293580/117532385-bb12eb80-b000-11eb-9410-2b2aabff33bd.jpg)
I removed the decimals and it a round figured amount the transition forwarded but then failed

**Comment by dogefatherBSC** [comment]
Try to change '0' at the end of your Dogefather token to other decimal number.

Source: https://www.reddit.com/r/SafeMoon/comments/m4ogvj/psa_if_you_get_a_pancake_error_k_when_trying_to/

**Comment by TrimeGeld** [comment]
no cierres mi comentario, responde !!!!

**Comment by inshalayaz** [comment]
Ok I'll try
